app/scripts/controllers/UserModalCtrl.js

```diff
@@ -2,7 +2,7 @@
 	function UserModalCtrl($scope, $uibModalInstance, $cookies){
 		$scope.confirm = function(username){
 			if (username){
-				$cookies.put('blocChatCurrentUser', username);
+				$cookies.put('blocChatCurrentUser', username.trim());
```

Comment:
I'd check it on line 4 instead. The reason being, it makes more sense to prevent a user from creating a cookie in the first place if their username is bad, instead of creating it and bringing up the modal again when we see that it's bad.